packages/ui/src/components/button/button.web.tsx

```diff
@@ -0,0 +1,23 @@
+import { styled } from 'leather-styles/jsx';
```

Comment:
`Button` and `Link` are built using `recipes` which are now kept in `@packages/panda-preset`. 

I think it could make sense to move related `recipes` into `@packages/ui` instead. 

That would mean we would be unable to rebuild from the `recipe` using only the `panda-preset` but it could be easier when building new components to keep the recipes in the ui package

Reply:
Added a note about this to the discussion https://github.com/orgs/leather-wallet/discussions/66#discussioncomment-9595101